### ISSUE TITLE
Input Helpers use 'value, not '_value

### ISSUE THREAD

**Issue by mslinn** [playframework/playframework#4396]
According to the [docs](https://www.playframework.com/documentation/2.4.x/ScalaCustomFieldConstructors) one might expect to pass the value of an input tag as `'_value -> "whatever"`, but instead `'value -> "whatever"` is what works.

**Comment by jroper** [comment]
Right, so we're using `_` as a namespacing feature to designate keys that have a special meaning to Play and won't end up being translated to attributes.  Maybe we should make that clear in the docs.  We do actually mention this [here](https://www.playframework.com/documentation/2.4.x/ScalaForms#Showing-forms-in-a-view-template):

> Note: All extra parameters will be added to the generated Html, unless they start with the _ character. Arguments starting with _ are reserved for field constructor arguments.

**Comment by mslinn** [comment]
"Nits 'R us!"
"In yer code, pickin' yer nits"
etc.
